scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,422 @@
+        default=DEFAULT_METADATA_DIR,
+        help="Output directory for releases metadata",
+    )
+    parser.add_argument(
+        "--quiet",
+        action="store_true",
+        help="Suppress output and only show errors",
+    )
+    parsed = parser.parse_args(args)
+
+    headers = get_headers()
+    with httpx.Client(headers=headers, timeout=None, follow_redirects=True) as client:
```

Ruff (0.15.15)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)